### PR TITLE
LPS-133436 | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-133436
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 1]
```diff
@@ -153,4 +153,134 @@ definition {
...
+			Navigator.gotoStagedSitePage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			Staging.gotoPublishToLive();
+
+			Staging.publishToLive();
+		}
+
+		task ("Assert all titles are shown in the live site") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:239]: View edited inline text of tabs at view mode in live site

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 2]
```diff
@@ -153,4 +153,134 @@ definition {
...
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:271]: View edited inline text of tabs at view mode in live site

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 3]
```diff
@@ -153,4 +153,134 @@ definition {
...
+				siteName = "Test Site Name Staging");
+		}
+
+		task ("Add a Tabs (Copy) to page") {
+			PageEditor.addFragment(
+				collectionName = "Collection Name",
+				fragmentName = "Tabs (Copy)");
+		}
+
+		task ("Edit inline text of Tabs") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:218]: Edit inline text of each tab

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 4]
```diff
@@ -153,4 +153,134 @@ definition {
 			value1 = '''<div class="fragment_1">Fragment Name</div>''');
 	}
 
+	@description = "This is a test case for LPS-129852. Enabled Propagate Fragment changes should not affect Collection Fragment in Staging and Live Site."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:156]: View the changes of fragment on content page still persist in staging and live site after overwrite the fragment

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 5]
```diff
@@ -153,4 +153,134 @@ definition {
...
+	@priority = "4"
+	test ViewChangesOfFragmentOnContentPagePersistWithEnabledPropagateFragmentInStagingLiveSite {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:158]: ViewChangesOfFragmentOnContentPagePersistInStagingAndLiveSiteAfterOverwriteTheFragment

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 6]
```diff
@@ -153,4 +153,134 @@ definition {
...
+				PageEditor.editFragmentText(
+					fragmentName = "Tabs (Copy)",
+					id = "title${n}",
+					text = "Test ${n}");
+			}
+
+			PageEditor.clickPublish();
+		}
+
+		task ("Publish the site to live") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:229]: Publish to Live via page

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 7]
```diff
@@ -153,4 +153,134 @@ definition {
...
+
+		task ("Import the fragment collection") {
+			Navigator.gotoStagedView();
+
+			FragmentsAdminNavigator.openFragmentsAdmin(siteURLKey = "test-site-name-staging");
+
+			FragmentsAdmin.importCollectionFromTempFolder();
+		}
+
+		task ("Publish the site to live") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:261]: Publish to Live via Staging admin

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 8]
```diff
@@ -153,4 +153,134 @@ definition {
...
+			Navigator.gotoStagedSitePage(
+				pageName = "Test Page Name",
+				siteName = "Test Site Name");
+
+			Staging.gotoPublishToLive();
+
+			Staging.publishToLive();
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:268]:
```suggestion
			Staging.openStagingAdmin(siteURLKey = "test-site-name-staging");

			Staging.publishCustomPublication();

```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase [hunk 9]
```diff
@@ -153,4 +153,134 @@ definition {
 			value1 = '''<div class="fragment_1">Fragment Name</div>''');
 	}
 
+	@description = "This is a test case for LPS-129852. Enabled Propagate Fragment changes should not affect Collection Fragment in Staging and Live Site."
+	@priority = "4"
+	test ViewChangesOfFragmentOnContentPagePersistWithEnabledPropagateFragmentInStagingLiveSite {
+		task ("Enable Propagate Fragment Changes Automatically in System Settings") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsWithStaging.testcase:159]: Add skip portal instance property

**Comment by ozysouza** [comment]
Hi @Tim-Cao PR updated with your recommendation.

**Comment by Tim-Cao** [comment]
See changes https://github.com/liferay-echo/liferay-portal/compare/ec9891a734d25fefe3572b09748eb698d1c20540...afa9a9f994575da0ee284b174d9cd6ca1ce964b2
https://github.com/liferay-echo/liferay-portal/pull/4941

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**